src/main/java/org/fenixedu/academic/ui/struts/action/coordinator/DegreeSiteManagementDispatchAction.java

```diff
@@ -137,16 +144,38 @@ public ActionForward viewDescriptionCurricularPlan(ActionMapping mapping, Action
             return new ActionForward(mapping.getInput());
         }
         request.setAttribute("degreeCurricularPlan", degreeCurricularPlan);
+        request.setAttribute(
+                "degreeCurricularPlanDescription",
+                new LocalizedString().with(Locale.ENGLISH, degreeCurricularPlan.getDescriptionEn()).with(new Locale("pt", "PT"),
```

Comment:
I would suggest using Locale.ENGLISH and Locale.getDefault() to drop the dependency of "Portuguese"